README.md

```diff
@@ -1,47 +1,44 @@
+[![Build Status](https://travis-ci.org/ember-learn/guides-source.svg?branch=master)](https://travis-ci.org/ember-learn/guides-source)
+[![This project is using Percy.io for visual regression testing.](https://percy.io/static/images/percy-badge.svg)](https://percy.io/Ember/guides-app)
 
-## Ember Guides Source [![Build Status](https://travis-ci.org/ember-learn/guides-source.svg?branch=master)](https://travis-ci.org/ember-learn/guides-source)
+# Ember Guides Source
 
-This repository contains the written content
-for the [Ember.js Guides](https://guides.emberjs.com).
-Here, contributors can file issues and submit PRs to
-help improve the learning experience of other developers.
+This repository contains the written content for the [Ember.js Guides](https://guides.emberjs.com). Contributors can file issues and submit pull requests (PRs) to help improve everyone's learning experience.
 
-Looking for repositories for the other parts of [emberjs.com](https://emberjs.com)?
-Check out
+Looking for repositories for the other parts of [emberjs.com](https://emberjs.com)? Check out
 [website](https://github.com/ember-learn/ember-website),
 [ember-api-docs](https://github.com/ember-learn/ember-api-docs),
 [super-rentals tutorial](https://github.com/ember-learn/super-rentals),
 [statusboard](https://github.com/ember-learn/statusboard),
 [deprecation-app](https://github.com/ember-learn/deprecation-app),
 and [styleguide](https://github.com/ember-learn/ember-styleguide).
 
+
 ## Contributing
 
-Welcome and thanks for your help! Please see [CONTRIBUTING.md](CONTRIBUTING.md)
-for detailed instructions on how to format your work and submit a Pull Request.
-First-time contributors are encouraged to choose issues that are labeled
-"help wanted" or "good for new contributors." If you have questions or
-want a buddy to pair with, you can join the
-[dev-ember-learning channel](https://discordapp.com/channels/480462759797063690/480777444203429888)
-in the [Ember Community Discord](https://discordapp.com/invite/zT3asNS).
+Welcome and thanks for your help!
+
+First-time contributors are encouraged to look at issues that are labeled **help wanted** or **good first issue**. If you have questions or want a buddy to pair with, you can join the [#dev-ember-learning channel](https://discordapp.com/channels/480462759797063690/480777444203429888) in the [Ember Community Discord](https://discordapp.com/invite/zT3asNS).
+
+Please see [CONTRIBUTING.md](CONTRIBUTING.md) for additional instructions on how to format your work and submit a PR.
 
 
 ## Project layout
 
-The Guides content takes the form of Markdown files (just like most READMEs).
-Each minor version of Ember has its own directory within `/guides/`.
-Pull requests should make edits to only the contents of the `guides/release` directory,
-which is the latest deployed version of the guides.
-The exception is that PRs to fix broken links in older versions of the guides are ok.
```

Comment:
Do we no longer want to let people know about this exception?

Reply:
My bad, the diffs make it hard to find a content after change. Line 30, after the changes, shows the exception for broken links. I think that's the exception that you were referring to, right?